src/gamepad/linux/linux-raw-gamepad.vala

```diff
@@ -0,0 +1,188 @@
+// This file is part of GNOME Games. License: GPLv3
+
+const int GUID_LENGTH = 8;
```

Comment:
It' probably good to explain why it's outside, with a FIXME so we remember to fix it if valac is fixed, maybe add something like:
// FIXME This should be in LinuxRawGamepad but a bug in valac prevent us from using it in 'requires' statements.